src/widgets/refinement-list/refinement-list.tsx

```diff
@@ -190,14 +192,14 @@ export type RefinementListWidgetParams = {
   /**
    * Templates to use for the widget.
    */
-  templates?: Partial<RefinementListTemplates>;
+  templates?: RefinementListTemplates;
   /**
    * CSS classes to add to the wrapping elements.
    */
   cssClasses?: RefinementListCSSClasses;
 };
 
-export const defaultTemplates: RefinementListOwnTemplates = {
+export const defaultTemplates: RefinementListWidgetTemplates = {
```

Comment:
We don't use the `WidgetTemplates` suffix elsewhere.

Reply:
Since it's only internal, does it make sense to you to have keep the naming with this implementation?https://github.com/algolia/instantsearch.js/pull/4785/commits/83c614ed93f39f4be32ad2ed7721b7de5b69ad56